src/frontends/onnx/frontend/src/op/reduce.hpp

```diff
@@ -29,6 +29,15 @@ ov::OutputVector reduce_l2(const ov::frontend::onnx::Node& node);
 namespace set_1 {
 ov::OutputVector reduce_max(const ov::frontend::onnx::Node& node);
 }  // namespace set_1
+namespace set_13 {
```

Comment:
This operator is also present in opset version 11. Is there any reason we're skipping version 11 ? In my case for ReduceLogSumExp there is no change between version 1 and 11, however maybe it would be good to include it for compatibility reasons ?

Reply:
Hi @mwilczy , I put a common motivation to do not introduce opset-11 for all Reduce* here https://github.com/openvinotoolkit/openvino/blob/c208a881cb4e1889d8d7126b838b4fcb86b44b8d/src/frontends/onnx/frontend/src/op/reduce.cpp#L167